integration-test/src/test/resources/trino-ci-testset/testsets/lakehouse-iceberg/00000_create_table.txt

```diff
@@ -6,7 +6,10 @@ CREATE TABLE
    name varchar,
    salary integer
 )
-COMMENT ''"
+COMMENT ''
+WITH (
+   location = 'hdfs://%/user/iceberg/warehouse/TrinoQueryIT/gt_db2.db/tb01'
```

Comment:
to fix this issue.
![image](https://github.com/datastrato/gravitino/assets/94670132/bfb64f39-2377-44ee-aaf9-b7e217a419d6)